src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1366,102 +1386,83 @@ data {
 "natural" parameters, but the model is otherwise identical.  In
-particular, the fairly diffuse priors on the coefficients and error
-scale are the same.  These could have been transformed as well, but
+particular, the fairly diffuse priors on the coefficients are the same.
+These could have been transformed as well, but
 here they are left as is, because the scales make sense as
-diffuse priors for standardized data; the priors could be made more
-informative.  For instance, because the outputs $y$ have been
-standardized, the error $\sigma$ should not be greater than 1, because
-that's the scale of the noise for predictors $\alpha = \beta = 0$.
+diffuse priors for standardized data.
 
 The original regression
 $$
 y_n
 = \alpha + \beta x_n + \epsilon_n
 $$
 has been transformed to a regression on the standardized variables,
+
 $$
-\textrm{z}_y(y_n)
-= \alpha'
-+ \beta' \textrm{z}_x(x_n)
-+ \epsilon'_n.
+y_n
+= \alpha' + \beta' \textrm{z}_x(x_n) + \epsilon_n.
 $$
-The original parameters can be recovered with a little algebra,
-\begin{align*}
-y_n &= \textrm{z}_y^{-1}(\textrm{z}_y(y_n)) \\
-    &= \textrm{z}_y^{-1}
-\left( \alpha' + \beta' \textrm{z}_x(x_n) + \epsilon_n' \right) \\
-    &= \textrm{z}_y^{-1}
-\left( \alpha' + \beta' \left( \frac{x_n - \bar{x}}{\texttt{sd}(x)} \right) + \epsilon_n' \right) \\
-    &= \texttt{sd}(y)
-\left( \alpha' + \beta' \left( \frac{x_n - \bar{x}}{\texttt{sd}(x)} \right) + \epsilon_n' \right) + \bar{y} \\
-    &=
-\left( \texttt{sd}(y) \left( \alpha' - \beta' \frac{\bar{x}}{\texttt{sd}(x)} \right) + \bar{y} \right)
-+ \left( \beta' \frac{\texttt{sd}(y)}{\texttt{sd}(x)} \right) x_n
-+ \texttt{sd}(y) \epsilon'_n,
-\end{align*}
+
 from which the original scale parameter values can be read off,
-$$
+
+
+\begin{align*}
 \alpha
-=
-\texttt{sd}(y)
-      \left(
-          \alpha'
-          - \beta' \frac{\bar{x}}{\texttt{sd}(x)}
-      \right)
-  + \bar{y};
-\qquad
-\beta = \beta' \frac{\texttt{sd}(y)}{\texttt{sd}(x)};
+= \alpha'
+-
+\beta' \frac{\bar{x}}{\texttt{sd}(x)};
 \qquad
-\sigma = \texttt{sd}(y) \sigma'.
+\beta = \frac{\beta'}{\texttt{sd}(x)};
+\end{align*}
+
+The independent priors on alpha and beta map to a multivariate normal prior on alpha_std and beta_std. 
+
+$$
+Cov(\alpha, \beta) = 0 \implies
+$$
+
+$$
+Cov(\alpha^\prime - \beta^\prime \frac{\bar{x}}{\text{sd}(x)}, \beta^\prime \frac{\bar{x}}{\text{sd}(x)}) = 0 \implies
+$$
+
+$$
+Cov(\alpha^\prime,\beta^\prime) = -\frac{\bar{x}}{\text{sd}(x)} Var(\beta^\prime)
 $$
 
 These recovered parameter values on the original scales can be
 calculated within Stan using a generated quantities block following
 the model block,
+
 ```stan
 generated quantities {
   real alpha;
   real beta;
   real<lower=0> sigma;
-  alpha = sd(y) * (alpha_std - beta_std * mean(x) / sd(x))
-           + mean(y);
-  beta = beta_std * sd(y) / sd(x);
-  sigma = sd(y) * sigma_std;
+  alpha = alpha_std - beta_std * mean_x / sd_x;
```

Comment:
Use compound declare/define for these.  I didn't check this math, but it looks a bit problematic.

Also, if you define `beta` first, you can use it in the definition of `alpha`.

I didn't follow why `alpha` is defined this way, so I hope it's OK.  Did you check this by running it to make sure you get the same results?

Reply:
I asked ChatGPT for the derivation.